### PR TITLE
Resources Page Acceptance Tests

### PR DESCRIPTION
**Pull request by LamboCreeper**
#### Overview
- Setup `codeceptjs` and `puppeteer` to run acceptance tests
    - Allows us to easily spot changes (we need to upgrade our Next.js version)
- Refactored original tests to be `test/unit/...`

### CODE REVIEW

File: .github/workflows/acceptance.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: CI
+on:
+  schedule:
+    - cron: "0 0 * * *"
+jobs:
```
Comment (LamboCreeper) [inline .github/workflows/acceptance.yml:5]: Should we also run the acceptance tests on pull requests, I guess we'd have to spin up the next.js server within CI?

File: test/acceptance/ResourcesTest.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Feature("Resources Page");
+
+Scenario("Hides Results Which Do Not Match Search Query", ({ I }) => {
+	I.amOnPage("https://codesupport.dev/resources");
```
Comment (Jason2605) [inline test/acceptance/ResourcesTest.js:4]: Are these not going to run on live?

What happens if we open a PR to change something, how would the tests reflect this?

Comment (LamboCreeper) [reply test/acceptance/ResourcesTest.js:4]: Yeah, valid point. I guess I was thinking they would let you know if the site broke, but it's a bit late then... Makes sense for them to run on the branch only?

Comment (Jason2605) [reply test/acceptance/ResourcesTest.js:4]: Yeah on the branch. Unsure how that will work in the CI/CD pipeline as we'd essentially need to spin up the current version (branch) and test - along with the backend, which doesn't want to be the live API

Comment (LamboCreeper) [reply test/acceptance/ResourcesTest.js:4]: I'm guessing we just use the output of the `build` step and run it against that, seems like we can mock the API via Codecept (see [docs](https://codecept.io/puppeteer/#mocking-requests)). I think doing that is fine given we’re testing frontend functionality not backend responses?

Comment (Jason2605) [reply test/acceptance/ResourcesTest.js:4]: Sounds like a good plan 👍

**Comment by LamboCreeper** [comment]
Closing due to inactivity.

**Pull request closed without merging**